src/test/java/it/mulders/mcs/search/printer/CoordinatePrinterTest.java

```diff
@@ -93,4 +98,50 @@ void should_print_snippet(CoordinatePrinter printer, String expected, SearchResp
 
         assertThat(xml).isEqualToIgnoringWhitespace(expected);
     }
+
+    @Nested
+    @DisplayName("CoordinatePrinter with vulnerabilities")
+    class CoordinatePrinterWithMultipleVulnerabilitiesTest {
+        private static final SearchQuery QUERY_DEPENDENCY_WITH_VULNERABILITIES =
+            SearchQuery.search("org.apache.shiro:shiro-web").build();
+        private static final SearchResponse.Response RESPONSE_WITH_VULNERABILITIES =
+            new SearchResponse.Response(1, 0, new SearchResponse.Response.Doc[]{
+                new SearchResponse.Response.Doc(
+                    "org.apache.shiro:shiro-web:1.9.0",
+                    "org.apache.shiro",
+                    "shiro-web",
+                    "1.9.0",
+                    null,
+                    "jar",
+                    1630022910000L,
+                    new ComponentReport(
+                        "pkg:maven/org.apache.shiro/shiro-web@1.9.0",
+                        "https://ossindex.sonatype.org/component/pkg:maven/org.apache.shiro/shiro-web@1.9.0?utm_source=postmanruntime&utm_medium=integration&utm_content=7.32.3",
```

Comment:
Could you please remove `?utm_source` and everything after it? It troubles the view a bit, and it's not relevant for the test. Applies to a few more cases in this file.